crates/db_views_moderator/src/mod_hide_community_view.rs

```diff
@@ -0,0 +1,69 @@
+use diesel::{result::Error, *};
+use lemmy_db_schema::{
+  limit_and_offset,
+  newtypes::CommunityId,
+  schema::{community, mod_hide_community, person},
+  source::{
+    community::{Community, CommunitySafe},
+    moderator::ModHideCommunity,
+    person::{Person, PersonSafe},
+  },
+  traits::{ToSafe, ViewToVec},
+};
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Serialize, Deserialize, Clone)]
+pub struct ModHideCommunityView {
+  pub mod_hide_community: ModHideCommunity,
+  pub admin: PersonSafe,
+  pub community: CommunitySafe,
+}
+
+type ModHideCommunityViewTuple = (ModHideCommunity, PersonSafe, CommunitySafe);
+
+impl ModHideCommunityView {
```

Comment:
@dessalines in the previous code review here you said it was missing admin_id around this part. What did you mean by that?

Reply:
Check out the other mod views, the parameters for the `list` function.

Reply:
I see